iina/Equalizations.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Untitled.swift
+//  iina
+//
+//  Created by Yuze Jiang on 2024/06/29.
+//  Copyright © 2024 lhc. All rights reserved.
+//
+
+class EQProfile: Codable {
+  var gains = [Double](repeatElement(0.0, count: 10))
+
+  init(_ values: [Double]) {
+    gains = values
+  }
+
+  init(fromCurrentSliders sliders: [NSSlider]) {
+    gains = sliders.map { $0.doubleValue }
+  }
+}
+
+class PresetEQProfile: EQProfile {
+  let localizationKey: String
+  var name: String {
+    NSLocalizedString(localizationKey, comment: localizationKey)
+  }
```

Comment:
Use `lazy var` so it can be cached.

(Why make it computed? Why not setting the name directly in `init()`?)